api/api.py

```diff
@@ -44,13 +50,22 @@ def too_large(e):
 
 
 class RecrudescenceTest(Resource):
+
+    decorators = [
+     limiter.limit("250/hour", error_message="Requests per hour limit exceeded"),
+     limiter.limit("5/minute", error_message="Requests per minute limit exceeded"),
+     limiter.limit("1/second", error_message="Requests per second limit exceeded")
+     ]
     def post(self):
         '''
-        Takes in a posted CSV data file, analyzes it to determine if each
+        Takes in a posted xlsx data file, analyzes it to determine if each
         case is a recrudescence/reinfection, and returns the calculated results
         '''
         uploaded_file = request.files.get('file')
-        iterations = request.args.get('iterations', default=10000, type=int)
+        iterations = request.args.get('iterations', default=50, type=int)
```

Comment:
Change the default back to 10000 (this should be set on the frontend, but might as well do it for future API use)

Reply:
Done. Of all things to forget to do...